lint/lint-gradle/src/main/java/androidx/lint/gradle/DiscouragedGradleMethodDetector.kt

```diff
@@ -242,90 +298,99 @@ class DiscouragedGradleMethodDetector : Detector(), Detector.UastScanner {
 
         // A map from eager method name to the containing class of the method and the name of the
         // replacement method, if there is a direct equivalent.
-        private val REPLACEMENTS =
+        private val REPLACEMENTS: Map<String, Replacement> =
```

Comment:
The fact that we now combine the fully qualified class name, the method name, and optionally argument into a single thing makes the RELEVANT_METHOD_NAMES parameter construction fragile.

I think we should move to using a real class here to reduce changes of mess ups / typos, etc